.drone.yml

```diff
@@ -121,20 +124,59 @@ pipeline:
       when:
         event: deployment
         environment: qa
-  
-    deploy-to-prod:
+        kube_namespace: cs-qa
+
+    deploy-to-qa-wcs:
+      image: quay.io/ukhomeofficedigital/kd:v1.16.0
+      environment:
+        - KUBE_NAMESPACE=wcs-qa
+        - ENVIRONMENT=qa
+        - KUBE_SERVER=https://kube-api-notprod.notprod.acp.homeoffice.gov.uk
+      secrets:
+        - hocs_frontend_qa_wcs
+        - poise_whitelist
+      commands:
+        - source version.txt
+        - echo $VERSION
+        - cd kube-hocs-frontend
+        - ./deploy.sh
+      when:
+        event: deployment
+        environment: qa
+        kube_namespace: wcs-qa
+
+    deploy-to-prod-cs:
       image: quay.io/ukhomeofficedigital/kd:v1.16.0
       environment:
+        - KUBE_NAMESPACE=cs-prod
         - ENVIRONMENT=prod
         - KUBE_SERVER=https://kube-api-prod.prod.acp.homeoffice.gov.uk
       secrets:
-        - hocs_frontend_prod
+        - hocs_frontend_prod_cs
         - poise_whitelist
       commands:
         - cd kube-hocs-frontend
         - ./deploy.sh
       when:
         event: deployment
         environment: prod
-  
-  
+        kube_namespace: cs-prod
```

Comment:
Don't forget to update the release documentation!

Reply:
The release process and commands do not change.